docs/source/install/uninstall.rst

```diff
@@ -113,6 +103,12 @@ below. Only execute the instructions for your distribution.
     sudo yum erase st2 st2mistral st2chatops st2web st2python bwc-ui st2flow
 
 
```

Comment:
Should we have a note here for RHEL/CentOS 8.x? 

I think it would be: 
 sudo yum erase st2 st2chatops st2web st2python bwc-ui st2flow

cc @armab

Reply:
@punkrokk  Is that not covered by the note below on line 106- to omit st2mistral?
That was the wording we had on the 3.2 documentation to cover RH8/Ubuntu 18.

HOWEVER I'm not sure if it was deliberate but some of the changes in the 3.2 documentation aren't on the master branch - so the notes that were added for 3.2 for RH 8 about missing mistral etc, seemed to have got dropped from master.

Reply:
Yes it's probably covered by the note. I'm a fan of having a nice `codeblock` to copy and paste, which is why I made the suggestion and it seemed in line with the rest of the page.

Reply:
@punkrokk Let me know if the changes made are preferred. I changed the note that is in the current st2docs (https://docs.stackstorm.com/install/uninstall.html) with code blocks like you suggested.

I noticed a few other updates needed, e.g.
st2mistral and postgres not used on ubuntu 18.
st2python not used on rh 7/8
So a few more changes, and gone for the code blocks.